0_RootFS/LLVMBootstrap@11/build_tarballs.jl

```diff
@@ -0,0 +1,12 @@
+# Include everything common between our LLVM versions (That's a lot)
+include("../llvm_common.jl")
+
+# Build the tarballs, then upload to Yggdrasil releases
+
+name = "LLVMBootstrap"
+version = v"11.0.1"
+sources, script, products, dependencies = llvm_build_args(;version=version)
+build_info = build_tarballs(ARGS, name, version, sources, script, [host_platform], products, dependencies;
+                            skip_audit=true, preferred_gcc_version=v"5.1")
```

Comment:
We don't actually have a GCCBootstrap 5.1 but do have 5.2. This just gets 5.2 and makes more sense in the context of this error:
```
[04:16:45] CMake Error at cmake/modules/CheckCompilerVersion.cmake:38 (message):
[04:16:45]   Host GCC version must be at least 5.1, your version is 4.8.5.
[04:16:45] Call Stack (most recent call first):
[04:16:45]   cmake/modules/CheckCompilerVersion.cmake:48 (check_compiler_version)
[04:16:45]   cmake/config-ix.cmake:13 (include)
[04:16:45]   CMakeLists.txt:655 (include)
```

Reply:
`preferred_gcc_version` isn't a strict requirement, there is a complex logic to choose the compiler version to use, this indicates the _preference_ to use GCC 5.1 or later (often when C++11 is needed)

Reply:
Ok, I just wanted to note why I chose 5.1 instead of 5.2